fiftyone/operators/registry.py

```
@@ -34,20 +32,25 @@ def get_operator(operator_uri, enabled=True):
             only disabled operators (False) or all operators ("all")
         type (None): whether to include only ``"panel"`` or ``"operator"`` type
             operators
+        builtins_only (False): whether to include only builtin operators
 
     Returns:
         a list of :class:`fiftyone.operators.Operator` instances
     """
     registry = OperatorRegistry(enabled=enabled)
-    return registry.list_operators(include_builtin=enabled != False, type=type)
+    return registry.list_operators(
+        include_builtin=enabled != False,
```

Ruff (0.8.2)

Avoid inequality comparisons to `False`; use `if enabled:` for truth checks

Replace with `enabled`

(E712)